benchalerts/parse_conbench.py

```diff
@@ -24,6 +24,38 @@ def _clean(text: str) -> str:
     return textwrap.fill(textwrap.dedent(text), 10000).replace("  ", "\n\n").strip()
 
 
+def benchmarks_with_errors(comparisons: List[RunComparison]) -> List[Tuple[str, str]]:
```

Comment:
```suggestion
def benchmarks_with_errors(comparisons: List[RunComparison]) -> List[Tuple[str, str, str, str]]:
```
I think?

Reply:
haha yep I had this change staged and forgot to push it 😄